pom.xml

```diff
@@ -3,20 +3,23 @@
   <parent>
     <groupId>org.jenkins-ci.plugins</groupId>
     <artifactId>plugin</artifactId>
-    <version>1.580</version>
+    <version>2.19</version>
   </parent>
 
   <artifactId>tfs-parent</artifactId>
   <packaging>pom</packaging>
   <name>Team Foundation Server Plug-in parent</name>
   <version>5.2.2-SNAPSHOT</version>
   <properties>
-    <jenkins.version>1.580</jenkins.version>
+    <jenkins.version>1.625.1</jenkins.version>
+    <java.level>7</java.level>
     <maven-surefire-plugin.version>2.19.1</maven-surefire-plugin.version>
     <!--  overriding old version from parent to be consistent with the above in this project -->
     <maven-surefire-report-plugin.version>${maven-surefire-plugin.version}</maven-surefire-report-plugin.version>
     <project.build.sourceEncoding>UTF-8</project.build.sourceEncoding>
     <project.reporting.outputEncoding>UTF-8</project.reporting.outputEncoding>
+    <maven-javadoc-plugin.version>2.8</maven-javadoc-plugin.version>
+    <findbugs.failOnError>false</findbugs.failOnError>
```

Comment:
Raised [JENKINS-40807](https://issues.jenkins-ci.org/browse/JENKINS-40807) to take care of them.